template/oss-npm-package/_package.json

```diff
@@ -34,13 +34,13 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@types/node": "^16.18.3",
+    "@types/node": "^20.11.0",
     "commitizen": "^4.2.4",
     "skuba": "*"
   },
   "packageManager": "pnpm@8.9.0",
   "engines": {
-    "node": ">=18.12"
```

Comment:
Does this deserve a changelog entry? We previously updated the .nvmrc / Dockerfiles to 20 already

Reply:
Package manifest is always iffy. You typically want them to lag one or two versions behind to support slower consumers